pages/wns.js

```diff
@@ -97,17 +81,18 @@ const Page = () => {
   const { registry, endpoint } = useRegistry(config);
 
   const handleRefresh = () => {
-    registry.getStatus()
-      .then(result => {
-        setStatus({ ts: Date.now(), result });
-
-        registry.queryRecords({ type })
-          .then(records => setRecords(records))
-          .catch(({ errors }) => setStatus({ ts: Date.now(), result, error: joinErrors(errors) }));
-      })
-      .catch(error => {
-        setStatus({ ts: Date.now(), error: error.statusText || 'Connection error.' });
-      });
+    // TODO(burdon): Failing.
+    // registry.getStatus()
```

Comment:
Config is pointing to https://wns-testnet.dev.wireline.ninja/graphql, which doesn't have the latest status API changes.

There is a bigger problem here. We're running start/stop/logs on the local WNS instance, but fetching records from https://wns-testnet.dev.wireline.ninja/graphql. If we're going to be pointing at a node we don't own, we should disable things like start/stop/log.

Reply:
https://github.com/dxos/dashboard/issues/22#issuecomment-605809739